apis/Google.Cloud.Firestore/Google.Cloud.Firestore/SerializationContext.cs

```diff
@@ -24,11 +25,22 @@ namespace Google.Cloud.Firestore
     /// </summary>
     internal sealed class SerializationContext
     {
+        // Important note: if this ever contains more state than just the converters,
+        // e.g. a reference to the Database, then WithWarningLogger will need to be adjusted accordingly.
+        
+        private readonly IReadOnlyDictionary<BclType, IFirestoreInternalConverter> _customConverters;
```

Comment:
nit: place this *after* the static `Default` property?

Reply:
Ooh yes, definitely.